### PR TITLE
Style core element

### PR DESCRIPTION
**Pull request by zieeco**
In this milestone, I styled the app following the design and also made it responsive on mobile and desktop

### CODE REVIEW

File: src/components/books/bookInfo.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
-import Button from '@mui/material/Button';
+// import Button from '@mui/material/Button';
 import { useDispatch } from 'react-redux';
```
Comment (Cathella) [inline src/components/books/bookInfo.js]: - [ ] Kindly remove all commented out lines of code from the project. This helps in maintaining code readability

File: src/pages.css [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+  bottom: 0;
+  right: 0;
+  left: 0;
+  padding: 0 5%;
+}
+
+header h1 {
+  color: #0290ff;
+  font-size: 1rem;
+}
```
Comment (Cathella) [inline src/pages.css:31]: Overall, you have done a beautiful job in styling this project according to the provided design. You just need to make a few changes to make perfect.

| Project Design | Zeplin Design |
| --- | --- |
| image | bookstore |

- [ ] On comparing both designs, I think the shadow on the book component is unnecessary and can be removed
- [ ] Also the  form background is not meant to be white as seen in the original design.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @zieeco 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## HELLO @zieeco 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**